common/versioning/version_tasks.gradle

```diff
@@ -1,5 +1,12 @@
 def getVersionFile() {
-    return file('versioning/version.properties')
+    
+    File file = new File("$rootDir/common/common/versioning/version.properties")
+    if(file.exists()){
+        return file;
+    }else{
+        return new File("$rootDir/common/versioning/version.properties")
+    }
```

Comment:
This seems to do the same thing in both cases.  Am I missing something?

Reply:
I think so... the path is different.

Reply:
Yup.  So one will cover common as part of android-complete, and the other as part of common.  I wonder that this shouldn't be parameterized to work with all the other packages.

Reply:
I was consider writing a document to cover gradle project composability.  To be honest I think the design of this can be improved.  Again since the same code is in multiple projects i'll take this a feedback for my next update of everything referenced by Android-complete.  Would be interested to see an example of what you're thinking about in terms of prioritization.

Reply:
So if this becomes a reusable activity that you can just add to your flow, then $rootDir/$versionFile, I would thing, where versionFile is in the workflow instance definition.

Reply:
Yep.  We can design something better here.  Sounds like a fun hackathon project.